terraform/us-dev.tfvars

```diff
@@ -0,0 +1,254 @@
+      "couchbase.enable.certificate.verification" = "false",
+      "transforms"                                = "setPartition",
+      "producer.retries"                          = "2147483647",
+      "couchbase.env.timeout.kvTimeout"           = "10s",
+      "couchbase.replicate.to"                    = "NONE",
+      "couchbase.source.handler"                  = "com.netdocuments.connect.kafka.handler.source.NDSourceHandler",
+      "couchbase.bucket"                          = "directory",
+      "couchbase.flow.control.buffer"             = "16m",
+      "couchbase.password"                        = "ei38kah3#$,ejw2",
+      "transforms.setPartition.partitions"        = "6",
+      "couchbase.topic"                           = "directory-search",
+      "errors.retry.timeout"                      = "10000",
+      "couchbase.custom.handler.nd.key.regex"     = "^(DUCOT-|UG-|NG-|CA-).*",
+      "topics"                                    = "directory-search",
+      "errors.retry.delay.max.ms"                 = "100",
+      "couchbase.log.document.lifecycle"          = "false",
+      "couchbase.custom.handler.nd.output.format" = "cloudevent",
+      "couchbase.seed.nodes"                      = "dev-dircb08.ndlab.local,dev-dircb09.ndlab.local,dev-dircb10.ndlab.local",
+      "couchbase.enable.tls"                      = "true",
+      "retries"                                   = "2147483647",
+      "classloader.check.plugin.dependencies"     = "true",
+      "couchbase.stream.from"                     = "SAVED_OFFSET_OR_NOW",
+      "couchbase.username"                        = "msk",
+      "errors.tolerance"                          = "all"
+    }
+
+    cb-nmd-acl-stream = {
+      "connector.class"                           = "com.couchbase.connect.kafka.CouchbaseSourceConnector",
+      "couchbase.persistence.polling.interval"    = "100ms",
+      "couchbase.enable.hostname.verification"    = "false",
+      "couchbase.bootstrap.timeout"               = "10s",
+      "couchbase.custom.handler.nd.fields"        = "documents.1.docProps.id,envProps.url,envProps.acl,envProps.containingcabs",
+      "tasks.max"                                 = "1",
+      "transforms.setPartition.type"              = "com.netdocuments.connect.kafka.transforms.SetKafkaPartitionFromKey",
+      "producer.retries"                          = "2147483647",
+      "couchbase.enable.certificate.verification" = "false",
+      "transforms"                                = "setPartition",
+      "couchbase.env.timeout.kvTimeout"           = "10s",
+      "couchbase.replicate.to"                    = "NONE",
+      "couchbase.source.handler"                  = "com.netdocuments.connect.kafka.handler.source.NDSourceHandler",
+      "couchbase.bucket"                          = "nmd",
+      "couchbase.flow.control.buffer"             = "16m",
+      "couchbase.password"                        = "$${secretsmanager:lab/cb/msk:couchbase_password}",
```

Comment:
couchbase.password and couchbase.username need to come from secretsmanager so they will be different in every environment.